api/specs/common/schemas/project-v0.0.1.json

```diff
@@ -252,6 +252,21 @@
                 }
               }
             },
+            "inputsUnits": {
+              "type": "object",
+              "description": "values of input unit",
+              "patternProperties": {
```

Comment:
I have 2 questions here:
- why not include this in the ```inputs``` field?
- why no units in the outputs?

Reply:
1) We want to keep them separated. At the moment, only the value is stored in the ```intputs ``` and we didn't want to change that.
2) I think that the user is not supposed to change the output units.

Reply:
ok for now, but somehow if the units may possibly change  in terms of being for example degrees, or kelvin, this is also a payload. 
As for 2., what if the service generates humanly readable units? similar to when you use some ubuntu commands for showing the available space, it will choose GB, or TB, or MB... just a thought.

Reply:
all cleared. thanks!